client/app.js

```diff
@@ -84,4 +84,4 @@ porybox.service('io', function () {
   return io;
 });
 
-ng.bootstrap(document, ['porybox']);
+ng.bootstrap(document, ['porybox'], {strictDi: true});
```

Comment:
Minifying breaks angular's "guessing what dependencies you need based on your parameter names" magic, so we will need to specify the dependencies explicitly from now on. I turned on `strictDi` which enforces this even when the files aren't minified.